### PR TITLE
Cleanup raw arrays in LB

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: src/ck-ldb/BaseLB.h [hunk 1]
```diff
@@ -95,7 +94,7 @@ class BaseLB: public CBase_BaseLB
     std::vector<LDCommData> commData;	// communication data - edge list representation
 				// of the communication between objects
 
-    int *objHash;		// this a map from the hash for the 4 integer
+    std::vector<int> objHash;		// this a map from the hash for the 4 integer
```
Comment (stwhite91) [inline src/ck-ldb/BaseLB.h:97]: what is "4 integer" here?

Comment (stwhite91) [reply src/ck-ldb/BaseLB.h:97]: Ah nvm, just saw that the comment extends to the next line

**Review by Wingpad — APPROVED** [review]
This seems to be worthwhile cleanup to me. Minor comment, but there are some whitespace inconsistencies between the `for` loops touched by this PR. I see `for...\n{\n`, `for...{\n`, and single-line for's without braces with both a newline and no newline. Any particular reason for these differences (e.g. trying to match local style)?

**Review by evan-charmworks — APPROVED** [review]

**Comment by rbuch** [comment]
> This seems to be worthwhile cleanup to me. Minor comment, but there are some whitespace inconsistencies between the `for` loops touched by this PR. I see `for...\n{\n`, `for...{\n`, and single-line for's without braces with both a newline and no newline. Any particular reason for these differences (e.g. trying to match local style)?

Nope, I just hadn't fully polished the formatting. I've push a fixup that takes care of it and makes some other miscellaneous improvements.

**Comment by rbuch** [comment]
ChaNGa will be broken until https://github.com/N-BodyShop/changa/pull/98 is merged.

**Review by evan-charmworks — APPROVED** [review]

**Review by Wingpad — APPROVED** [review]

**Pull request merged**